### PR TITLE
Remove server ftcp code

### PR DESCRIPTION
**Pull request by achamayou**
Another sub-item from #767

### CODE REVIEW

**Comment by achamayou** [comment]
depends on #783

File: src/consensus/pbft/pbftconfig.h [hunk 1]
```diff
@@ -82,9 +82,6 @@ namespace pbft
 
       // TODO: HTTP signatures are not handled by PBFT
       // https://github.com/microsoft/CCF/issues/720
-#ifndef FTCP
```
Comment (jumaffre) [inline src/consensus/pbft/pbftconfig.h]: We still need this.

File: src/enclave/rpcclient.h [hunk 2]
```diff
@@ -1,35 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (jumaffre) [inline src/enclave/rpcclient.h]: We could move `HTTPClientEndpoint` in `httpendpoint.h` to its own file maybe?

**Comment by jumaffre** [comment]
Looks like the `member_client_test` is still failing with `AttributeError: 'Consortium' object has no attribute '_member_client_rpc_as_json'`

File: src/node/rpc/test/frontend_test.cpp [hunk 3]
```diff
@@ -508,26 +508,6 @@ TEST_CASE("process")
     CHECK(value.req.empty());
     CHECK(value.sig == signed_call[jsonrpc::SIG]);
   }
-
-  // TODO: verify_client_signature in HTTP
```
Comment (olgavrou) [inline src/node/rpc/test/frontend_test.cpp]: Has this TODO been implemented or ticketed?

**Comment by cimetrics** [comment]
remove_server_ftcp_code@[4544 aka 20200204.14](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4544&view=results) vs master ewma over 30 builds from [4131](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4131&view=results) to [4542](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4542&view=results)
images

File: src/node/rpc/test/frontend_test.cpp (hunk 3 above)
Comment (achamayou) [reply src/node/rpc/test/frontend_test.cpp]: I think it is, and looking at frontend.h, it seems to be, but I can see that some of the perf tests aren't enabled (sig forwarding for example), so I'll double check what's up there.

Comment (achamayou) [reply src/node/rpc/test/frontend_test.cpp]: So the check is implemented, but it fails with the C++ test client for some reason, and that's ticketed here: https://github.com/microsoft/CCF/issues/787

**Pull request merged**